docs/developers/node-api.md

```diff
@@ -679,3 +679,324 @@ Response
+            },
+            {
+              "key": "sender",
+              "value": "celestia14m95yh3u4tp7xr4cwq2l773lmdk9f0lfr3k87g"
+            },
+            {
+              "key": "amount",
+              "value": "1000utia"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "gas_wanted": 80000,
+  "gas_used": 73842,
+  "events": [
+    {
+      "type": "tx",
+      "attributes": [
+        {
+          "key": "ZmVl",
+          "value": null,
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "tx",
+      "attributes": [
+        {
+          "key": "YWNjX3NlcQ==",
+          "value": "Y2VsZXN0aWExNG05NXloM3U0dHA3eHI0Y3dxMmw3NzNsbWRrOWYwbGZyM2s4N2cvMTE1",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "tx",
+      "attributes": [
+        {
+          "key": "c2lnbmF0dXJl",
+          "value": "VlVmaDR4V2RlbXN3Vjdoa05BTVR2V1ZYTWRGRkhLbzBpRGNDenMyQmpvOUVkM1JJUnpjSHFHWk52RTNaUDUyS0tyaDhqTXJtbjEyYU1tODA3akVKZ2c9PQ==",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "message",
+      "attributes": [
+        {
+          "key": "YWN0aW9u",
+          "value": "L2Nvc21vcy5iYW5rLnYxYmV0YTEuTXNnU2VuZA==",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "coin_spent",
+      "attributes": [
+        {
+          "key": "c3BlbmRlcg==",
+          "value": "Y2VsZXN0aWExNG05NXloM3U0dHA3eHI0Y3dxMmw3NzNsbWRrOWYwbGZyM2s4N2c=",
+          "index": true
+        },
+        {
+          "key": "YW1vdW50",
+          "value": "MTAwMHV0aWE=",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "coin_received",
+      "attributes": [
+        {
+          "key": "cmVjZWl2ZXI=",
+          "value": "Y2VsZXN0aWExNnF4NzMwNnZlaHl5OW5kdHdzeTZjcmttZnBmN2VubXBodmwyZTI=",
+          "index": true
+        },
+        {
+          "key": "YW1vdW50",
+          "value": "MTAwMHV0aWE=",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "transfer",
+      "attributes": [
+        {
+          "key": "cmVjaXBpZW50",
+          "value": "Y2VsZXN0aWExNnF4NzMwNnZlaHl5OW5kdHdzeTZjcmttZnBmN2VubXBodmwyZTI=",
+          "index": true
+        },
+        {
+          "key": "c2VuZGVy",
+          "value": "Y2VsZXN0aWExNG05NXloM3U0dHA3eHI0Y3dxMmw3NzNsbWRrOWYwbGZyM2s4N2c=",
+          "index": true
+        },
+        {
+          "key": "YW1vdW50",
+          "value": "MTAwMHV0aWE=",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "message",
+      "attributes": [
+        {
+          "key": "c2VuZGVy",
+          "value": "Y2VsZXN0aWExNG05NXloM3U0dHA3eHI0Y3dxMmw3NzNsbWRrOWYwbGZyM2s4N2c=",
+          "index": true
+        }
+      ]
+    },
+    {
+      "type": "message",
+      "attributes": [
+        {
+          "key": "bW9kdWxl",
+          "value": "YmFuaw==",
+          "index": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+## Helpful CLI Commands
```

Comment:
I'd probably move this section to a separate doc.

We are probably going to eventually phase out this doc for Node API reference when OpenRPC implementation is ready.

Reply:
sounds good, will add a page for "helpful CLI commands" for node, similar to app